### PR TITLE
Joe additions

### PR DESCRIPTION
**Pull request by jojonium**
Added ralph's suggestions

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -1964,6 +1968,12 @@ \section{Conclusion and Future Work}
 focused only on the features that made it into the final app from the start we
 could have had more time to polish and refine those features.
 
+On a similar note, the requirements and scope of the project changed greatly
+from week to week, and sometimes even within one day. We spent a lot of time
```
Comment (bandaloo) [inline paper.tex]: sort of don't like the term "a lot" but i could go either way

File: paper.tex [hunk 2]
```diff
@@ -1972,8 +1982,43 @@ \section{Conclusion and Future Work}
 Kinkaku-ji. More extensive user testing at the actual site could have helped us
 catch bugs earlier and lead to a more polished final product.
 
-Despite these issues our team is happy with the product we delivered to
-Kyoto~VR. Selling ARuko on app stores was not within the scope of our academic
+While not a disaster, we had some initial trouble getting started with Unity.
+Different versions of Unity do not play well with each other, and it often runs
+into bugs on older hardware, such as the laptops we had to work with.
+Furthermore, a stable version of the Unity editor is not available on
+GNU/Linux~\autocite{best2019}, so one of our team members had to install MacOS
```
Comment (bandaloo) [inline paper.tex]: pretty sure officially, it is capitalized macOS, even according to Wikipedia

**Review by bandaloo — CHANGES_REQUESTED** [review]

File: paper.tex (hunk 1 above)
Comment (jojonium) [reply paper.tex]: Replaced with "significant amount of time"

File: paper.tex (hunk 2 above)
Comment (jojonium) [reply paper.tex]: Fixed

File: paper.tex [hunk 3]
```diff
@@ -205,37 +206,71 @@ \section{Introduction}
...
 complete platform for designing and experiencing AR-enhanced tours. Editour
 makes it easy to design custom audio tours including audio, text, and images,
 while ARuko allows users to seamlessly experience historic sites in an intuitive
-and novel way. The experience of walking through the site is enhanced by
-informative narration and AR pop-ups that do not alter the buildings or ground
-in real life.
+and novel way. Many AR apps require specific markers, like the those shown in
+Figure~\ref{fig:arMarker}. Too avoid needing to place any physical object that
+would disturb the buildings or grounds of a historic site our app does not rely
+on these markers. Instead, our ARuko recognizes images of objects that are
```
Comment (bandaloo) [inline paper.tex]: change "our ARuko" to "our app ARuko"

File: paper.tex [hunk 4]
```diff
@@ -302,17 +339,17 @@ \subsubsection{Augmented Reality Use Cases}
...
 
-Many of the existing applications using AR, in our eyes, do not leverage the
+Most existing applications using AR, in our team's opinions, do not leverage the
 full potential of this emerging technology. As with any new field, our
-understanding of AR and how to develop for it is limited. All sorts of
-different challenges and limitations face the development and production of AR
-apps. Here are some that we considered when working on our app.
+understanding of AR and how to develop for it is limited. All sorts of different
+challenges and limitations face the development and production of AR apps. Here
+are some that we considered when working on our app.
```
Comment (bandaloo) [inline paper.tex]: Maybe write "In this section, we discuss some of the challenges..." rather than "Here are some..."

File: paper.tex [hunk 5]
```diff
@@ -302,17 +339,17 @@ \subsubsection{Augmented Reality Use Cases}
...
 
 Many of the applications and use cases that exist for AR right now are, simply
 put, not very good. Many AR apps and features available on smartphones offer
-very little in the way of functionality or practicality; quite simply, they are
-primarily used for minor spectacles and simple
-tricks~\autocite{theappsolutions2018}. For example, one of the most popular
+very little in the way of functionality or practicality; quite simply, they
+primarily primarily revolve around deploying
```
Comment (bandaloo) [inline paper.tex]: "primarily primarily"

File: paper.tex [hunk 6]
```diff
@@ -1929,8 +1983,43 @@ \section{Conclusion and Future Work}
 Kinkaku-ji. More extensive user testing at the actual site could have helped us
 catch bugs earlier and lead to a more polished final product.
 
-Despite these issues our team is happy with the product we delivered to
-Kyoto~VR. Selling ARuko on app stores was not within the scope of our academic
+While not a disaster, we had some initial trouble getting started with Unity.
```
Comment (bandaloo) [inline paper.tex]: do we need "While not a disaster"

File: paper.tex [hunk 7]
```diff
@@ -1929,8 +1983,43 @@ \section{Conclusion and Future Work}
...
+Different versions of Unity do not play well with each other, and it often runs
```
Comment (bandaloo) [inline paper.tex]: change to "Different versions of Unity do not play well with each other. As a result, we often ran into bugs on older hardware..."

File: paper.tex [hunk 8]
```diff
@@ -1929,8 +1983,43 @@ \section{Conclusion and Future Work}
...
+
+Editour was not initially a requirement that Atticus had in mind, but it turned
+out to be a very useful tool in designing tours and allowing the app to be
+extensible. Two members of our team, Cole Granof and Joseph Petitti, had
+experience building web applications with HTML, CSS, and JavaScript, and the
+Editour allowed them to use these skills to help the project.
+
+Our team's broad range of skills made it so that each member had specific tasks
+that they were good at. This made the division of labor easier, and throughout
+the project we never had issues splitting up work.
```
Comment (bandaloo) [inline paper.tex]: Maybe this paragraph should go before the previous one?

File: paper.tex [hunk 9]
```diff
@@ -2043,11 +2132,10 @@ \section{Testing Survey Results}
 26\textsuperscript{th}, we had each test subject fill out a simple survey with
 their thoughts on the app. The full results are listed below.
 
-% for tabularx
-% \newcolumntype{L}{>{\raggedright\arraybackslash\footnotesize}X}%
-
 \begin{table}[h]
 \begin{singlespace}
+\renewcommand{\arraystretch}{2.0} % add space between rows
```
Comment (bandaloo) [inline paper.tex:2137]: so nice

**Review by bandaloo — APPROVED** [review]

File: paper.tex (hunk 3 above)
Comment (jojonium) [reply paper.tex]: Changed to just "ARuko"

File: paper.tex (hunk 4 above)
Comment (jojonium) [reply paper.tex]: Fixed

File: paper.tex (hunk 5 above)
Comment (jojonium) [reply paper.tex]: Fixed

File: paper.tex (hunk 6 above)
Comment (jojonium) [reply paper.tex]: Removed

File: paper.tex (hunk 7 above)
Comment (jojonium) [reply paper.tex]: Fixed

File: paper.tex (hunk 8 above)
Comment (jojonium) [reply paper.tex]: Moved

File: paper.tex (hunk 9 above)
Comment (jojonium) [reply paper.tex:2137]: thx

**Pull request merged**